packages/stark-ui/assets/styles/_variables.scss

```diff
@@ -151,6 +151,11 @@ $button-theme: (
     contrast: $light-primary-text
   )
 );
+$dropdown-theme: (
```

Comment:
We should have a more global setting for those colors, as they can be used in many components not only dropdown. 

I have created a ticket about it: https://github.com/NationalBankBelgium/stark/issues/662 so I will handle that later.